tests/Unit/PointwiseFunctions/AnalyticSolutions/EinsteinSolutions/Test_KerrSchild.cpp

```diff
@@ -0,0 +1,187 @@
+    }
+  }
+
+  const DataVector r = magnitude(x);
+  auto lapse_test = make_with_value<Scalar<DataVector>>(x, 0.0);
+  get<>(lapse_test) = 1.0 / sqrt(1.0 + 2.0 * mass / r);
+  CHECK_ITERABLE_APPROX(lapse, lapse_test);
+
+  auto shift_test =
+      make_with_value<tnsr::I<DataVector, 3, Frame::Inertial>>(x, 0.0);
+  for (size_t i = 0; i < 3; ++i) {
+    shift_test.get(i) =
+        2.0 * mass * x.get(i) / square(r) * square(get<>(lapse));
+  }
+  CHECK_ITERABLE_APPROX(shift, shift_test);
+
+  auto d_lapse_test =
+      make_with_value<tnsr::i<DataVector, 3, Frame::Inertial>>(x, 0.0);
+  for (size_t i = 0; i < 3; ++i) {
+    d_lapse_test.get(i) = mass * x.get(i) / cube(r) * cube(get<>(lapse));
+  }
+  CHECK_ITERABLE_APPROX(d_lapse, d_lapse_test);
+
+  auto g_test =
+      make_with_value<tnsr::ii<DataVector, 3, Frame::Inertial>>(x, 0.0);
+  for (size_t i = 0; i < 3; ++i) {
+    for (size_t j = i; j < 3; ++j) {
+      g_test.get(i, j) = 2.0 * mass * x.get(i) * x.get(j) / cube(r);
+    }
+    g_test.get(i, i) += 1.0;
+  }
+  CHECK_ITERABLE_APPROX(g, g_test);
+
+  auto dg_test =
+      make_with_value<tnsr::ijj<DataVector, 3, Frame::Inertial>>(x, 0.0);
+  for (size_t k = 0; k < 3; ++k) {
+    for (size_t i = 0; i < 3; ++i) {
+      for (size_t j = i; j < 3; ++j) {  // Symmetry
+        dg_test.get(k, i, j) =
+            -6.0 * mass * x.get(i) * x.get(j) * x.get(k) / cube(r) / square(r);
+        if (k == j) {
+          dg_test.get(k, i, j) += 2.0 * mass * x.get(i) / cube(r);
+        }
+        if (k == i) {
+          dg_test.get(k, i, j) += 2.0 * mass * x.get(j) / cube(r);
+        }
+      }
+    }
+  }
+  for (size_t k = 0; k < 3; ++k) {
+    for (size_t i = 0; i < 3; ++i) {
+      for (size_t j = i; j < 3; ++j) {  // Symmetry
+        CHECK_ITERABLE_APPROX(d_g.get(k, i, j), dg_test.get(k, i, j));
+      }
+    }
+  }
+}
+
+void test_einstein_solution() {
+  // Parameters
+  //   ...for KerrSchild solution
+  const double mass = 1.01;
+  const std::array<double, 3> spin = {{0.1, 0.2, 0.3}};
+  const std::array<double, 3> center = {{0.01, 0.02, 0.03}};
+  //   ...for grid
+  const size_t grid_size = 8;
+  const std::array<double, 3> lower_bound = {{1.99, -0.01, -0.01}};
+  const std::array<double, 3> upper_bound = {{2.01, 0.01, 0.01}};
+
+  EinsteinSolutions::KerrSchild solution(mass, spin, center);
+  verify_time_independent_einstein_solution(
+      solution, grid_size, lower_bound, upper_bound,
+      std::numeric_limits<double>::epsilon() * 100000);
+}
+}  // namespace
+
+SPECTRE_TEST_CASE(
+    "Unit.PointwiseFunctions.AnalyticSolutions.EinsteinSolution.KerrSchild",
+    "[PointwiseFunctions][Unit]") {
+  test_schwarzschild();
+  test_einstein_solution();
+}
+
+// [[OutputRegex, Spin magnitude must be < 1]]
+SPECTRE_TEST_CASE(
+    "Unit.PointwiseFunctions.AnalyticSolutions.EinsteinSolution.KerrSchildSpin",
+    "[PointwiseFunctions][Unit]") {
+  ERROR_TEST();
+  EinsteinSolutions::KerrSchild solution(1.0, {{1.0, 1.0, 1.0}},
```

Comment:
Should there also be tests for constructing via Options?

Reply:
Yes, there are now.